src/flag_gems/ops/sum.py

```diff
@@ -124,6 +124,13 @@ def sum_dim(inp, dim=None, keepdim=False, *, dtype=None):
         if dtype is torch.bool:
             dtype = torch.int64
 
+    if dim == []:
+        if not keepdim:
+            return sum(inp, dtype=dtype)
+        else:
+            dim_num = inp.ndim
+            return torch.reshape(sum(inp, dtype=dtype), [1] * dim_num)
```

Comment:
Can we simplify this code to `return torch.reshape(sum(inp, dtype=dtype), [1] * inp.ndim)` ?